fintech-examples/fintech-ui/src/app/guards/auth.guard.ts

```diff
@@ -9,11 +9,10 @@ export class AuthGuard implements CanActivate {
   constructor(private authService: AuthService) {}
 
   canActivate(): boolean {
-    const isLoggedIn = this.authService.isLoggedIn();
-
-    if (!isLoggedIn) {
+    const logged =  this.authService.isLoggedIn();
+    if (! logged) {
       this.authService.logout();
     }
-    return isLoggedIn;
+    return logged;
```

Comment:
Please revert. The previous naming is probably better

Reply:
yep, done